applications/matter_bridge/CMakeLists.txt

```
@@ -28,6 +28,7 @@ target_include_directories(app PRIVATE
     src/bridged_device_types
     ${ZEPHYR_NRF_MODULE_DIR}/samples/matter/common/src/bridge/
     ${ZEPHYR_NRF_MODULE_DIR}/samples/matter/common/src/bridge/util
+    ${ZEPHYR_NRF_MODULE_DIR}/samples/matter/persistent_storage
```

Does it even work? Path seems to be incorrect.
```suggestion
    ${ZEPHYR_NRF_MODULE_DIR}/samples/matter/common/src/persistent_storage
```

---

Yes, it's wrong yet redundant anyway because the common module pulls all necessary files by itself.